### PR TITLE
feat: update funder guide + add alert for advanced payout

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- updated the funder guide
- compressed images
- added alert for advanced payout

##### Refers/Fixes
closes https://github.com/gitcoinco/web/issues/4845
closes https://github.com/gitcoinco/web/issues/4680

##### Testing

screencapture-localhost-8000-bounty-quickstart-2019-08-06-01_35_25

Screenshot 2019-08-06 at 10 58 23 PM

Screenshot 2019-08-06 at 11 00 25 PM

### CODE REVIEW

File: app/dashboard/templates/quickstart.html [hunk 1]
```diff
@@ -32,123 +33,123 @@
...
           <img src="{% static 'v2/images/quickstart-box.svg' %}" />
         </div>
-        <div class="col-md-6 col-sm-12 p-5 steps-list">
+        <div class="col-md-7 col-sm-12 p-5 steps-list">
           <ol class="font-subheader">
             <li>{% trans "1) Create a Github Issue, you'll need the issue URL" %}</li>
             <li>{% trans "2) Select the type of issue to fund" %}</li>
             <li>{% trans '3) Complete the form and press "Fund Issue"' %}</li>
-            <li>{% trans "4) Confirm the transaction in metamask" %}</li>
+            <li>{% trans "4) Confirm the transaction in MetaMask" %}</li>
```
Comment (danlipert) [inline app/dashboard/templates/quickstart.html]: Nice catch

File: app/dashboard/views.py [hunk 2]
```diff
@@ -1737,8 +1737,13 @@ def funder_payout_reminder(request, bounty_network, stdbounties_id):
 
 
 def quickstart(request):
-    """Display quickstart guide."""
-    return TemplateResponse(request, 'quickstart.html', {})
+    """Display Quickstart Guide."""
+
+    from dashboard.context.quickstart import quickstart
```
Comment (danlipert) [inline app/dashboard/views.py]: Can we import this at the top of the file to make this response faster?

Comment (thelostone-mc) [reply app/dashboard/views.py]: Oh wouldn't that import it for every other view too / am i wrong in that assumption?

File: app/dashboard/views.py [hunk 3]
```diff
@@ -1737,8 +1737,13 @@ def funder_payout_reminder(request, bounty_network, stdbounties_id):
...
+    activities = Activity.objects.filter(activity_type='new_bounty').order_by('-created')[:5]
```
Comment (octavioamu) [inline app/dashboard/views.py]: do we need a network filter here?

File: app/dashboard/views.py (hunk 2 above)
Comment (thelostone-mc) [reply app/dashboard/views.py]: Updated it ^_^

**Pull request merged**